apps/web-client/src/routes/outbound/[id]/+page.svelte

```diff
@@ -603,6 +596,12 @@
 					</DropdownWrapper>
 				</div>
 			</div>
+			{#if no_warehouse_selected_commit_self.length}
```

Comment:
This takes up a lot of space and doesn't disappear, how about [the popover](https://www.melt-ui.com/docs/builders/popover) that hover's next to the commit button?